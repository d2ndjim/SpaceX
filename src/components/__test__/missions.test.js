import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../redux/configureStore';
import Missions from '../mission/missions';

describe('Missions test', () => {
  it('renders missions', () => {
    const app = renderer
      .create(
        <Provider store={store}>
          <Missions />
        </Provider>,
      )
      .toJSON();
    expect(app).toMatchSnapshot();
  });
});
