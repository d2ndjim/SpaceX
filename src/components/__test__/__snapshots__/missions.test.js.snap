// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Missions test renders missions 1`] = `
<table>
  <thead>
    <tr>
      <th>
        Mission
      </th>
      <th>
        Description
      </th>
      <th>
        Status
      </th>
      <th
        aria-label="Empty cell"
      />
    </tr>
  </thead>
  <tbody />
</table>
`;
